ConsoleApp/MyStack/MyStack.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp.MyStack
+{
+    internal class MyStack<T> : IEnumerable<T>
+    {
+        private Element<T> _head;
+        private int _count;
+
+        public int GetCount
+        {
+            get { return _count; }
+            set { }
+        }
+
+        public MyStack()
+        {
+            _head = null;
+            _count = 0;
+        }
+        public MyStack(MyStack<T> stack) : this()
+        {
+            foreach (T item in stack)
+            {
+                Push(item);
+            }
+        }
+        public bool isEmpty() => GetCount == 0 ? true : false;
```

Comment:
code convention violated here
```suggestion
        public bool IsEmpty() => GetCount == 0 ? true : false;
```